modules/contrib/better_exposed_filters/README.txt

```diff
@@ -0,0 +1,34 @@
+-- SUMMARY --
```

Comment:
Contrib modules should not be added to the repository. They should be specified in drush make and composer files.